instrumentation/opentelemetry-instrumentation-click/src/opentelemetry/instrumentation/click/__init__.py

```diff
@@ -74,6 +94,12 @@ def _command_invoke_wrapper(wrapped, instance, args, kwargs, tracer):
         return wrapped(*args, **kwargs)
 
     ctx = args[0]
+
+    # we don't want to create a root span for long running processes like servers
+    # otherwise all requests would have the same trace id
```

Comment:
Would it make sense to also add this to the docs above? It might be noise to newer users but could help as this relatively new instrumentor might evolve.